### ISSUE TITLE
Question : How to jump to exact search word position after perform a blines command? 

### ISSUE THREAD

**Issue by xzbdmw** [ibhagwan/fzf-lua#1044]
### Description 
Now the blines command only jumps to the beginning of the line, why can't it jump to the column position of the exact character I searched for. Since its name is called *blines* but I can't find a command to fzf word.

**Comment by ibhagwan** [comment]
`blines` has no concept of column, it’s fuzzy matching it cannot guess what you meant after selection, if up’s want an exact match with columns why don’t you use `lgrep_curbuf`, you can always `ctrl-g` to refine the matches with fuzzy.

**Comment by ibhagwan** [comment]
You can also create a custom default action and search for the last query on the selected line but as I wrote, if it’s based on a fuzzy match you might not find anything.

**Comment by xzbdmw** [comment]
> `blines` has no concept of column, fuzzy matching it cannot guess what you meant post selection, if you want an exact match with columns why don’t you use `lgrep_curbuf`, you can always `ctrl-g` to refine the matches with fuzzy.

Switching from grep to fuzzy will cause the pointer to stay at the position before pressing C-g and there will be no precise position jump, like the screenshot below.
I first use grep to search for bg, and then the cursor stays at bg(in preview), which is an exact positional match. Then, when I switch to fuzzy search for fg, the cursor in the preview window still stays at bg, and even after I jump over. Is this just the expected behavior?

https://github.com/ibhagwan/fzf-lua/assets/97848247/ff46d3a0-813c-449c-b4c3-284528485985

**Comment by ibhagwan** [comment]
Yes, this is expected, the cursor position is determined by the line/column numbers output of rg, the fuzzy matching does not change cursor position of the preview, the entry is in the format of `file:line:col:text` so `file:1:1` will set the cursor at 1:1 regardless of what the fuzzy match finds.

**Comment by ibhagwan** [comment]
Btw, the reason you’re not seeing the file name is because the `curbuf` pickers hide it, the underlying command is the output of `rg’ exactly as `live_grep`.

**Comment by xzbdmw** [comment]
Telescope current_buffer_fuzzy can jump to exact match, but its workspace fuzzy search is buggy, so I hope this plugin can have a option to do the same:P
